src/osdc/Objecter.h

```diff
@@ -2016,33 +2018,64 @@ class Objecter : public md_config_obs_t, public Dispatcher {
+      "objecter_mclock_service_tracker")),
+    m_objecter_inject_no_watch_ping(cct->_conf->get_val<bool>(
+      "objecter_inject_no_watch_ping")),
+    m_objecter_debug_inject_relock_delay(cct->_conf->get_val<bool>(
+      "objecter_debug_inject_relock_delay")),
+    m_osdc_blkin_trace_all(cct->_conf->get_val<bool>(
+      "osdc_blkin_trace_all")),
+    m_objecter_tick_interval(cct->_conf->get_val<double>(
+      "objecter_tick_interval")),
+    m_objecter_timeout(cct->_conf->get_val<double>(
+      "objecter_timeout")),
+    m_osd_client_op_priority(cct->_conf->get_val<uint64_t>(
+      "osd_client_op_priority")),
+    m_objecter_inflight_op_bytes(cct->_conf->get_val<uint64_t>(
+      "objecter_inflight_op_bytes")),
+    m_objecter_inflight_ops(cct->_conf->get_val<uint64_t>(
+      "objecter_inflight_ops")),
+    m_objecter_retry_writes_after_first_reply(cct->_conf->get_val<bool>(
+      "objecter_retry_writes_after_first_reply")),
     trace_endpoint("0.0.0.0", 0, "Objecter"),
-    mclock_service_tracker(cct->_conf->objecter_mclock_service_tracker),
+    mclock_service_tracker(m_objecter_mclock_service_tracker),
     osdmap(new OSDMap),
     max_linger_id(0),
-    keep_balanced_budget(false), honor_osdmap_full(true), osdmap_full_try(false),
+    keep_balanced_budget(false),
+    honor_osdmap_full(true), osdmap_full_try(false),
     blacklist_events_enabled(false),
     last_seen_osdmap_version(0), last_seen_pgmap_version(0),
-    logger(NULL), tick_event(0), m_request_state_hook(NULL),
+    logger(nullptr), tick_event(0), m_request_state_hook(nullptr),
     homeless_session(new OSDSession(cct, -1)),
     mon_timeout(ceph::make_timespan(mon_timeout)),
     osd_timeout(ceph::make_timespan(osd_timeout)),
-    op_throttle_bytes(cct, "objecter_bytes",
-		      cct->_conf->objecter_inflight_op_bytes),
-    op_throttle_ops(cct, "objecter_ops", cct->_conf->objecter_inflight_ops),
+    op_throttle_bytes(cct, "objecter_bytes", m_objecter_inflight_op_bytes),
+    op_throttle_ops(cct, "objecter_ops", m_objecter_inflight_ops),
     epoch_barrier(0),
-    retry_writes_after_first_reply(cct->_conf->objecter_retry_writes_after_first_reply)
+    retry_writes_after_first_reply(m_objecter_retry_writes_after_first_reply)
   {
-    if (cct->_conf->objecter_mclock_service_tracker) {
+    if (m_objecter_mclock_service_tracker)
       qos_trk = ceph::make_unique<dmc::ServiceTracker<int>>();
-    }
   }
   ~Objecter() override;
 
   void init();
   void start(const OSDMap *o = nullptr);
   void shutdown();
 
+  // ------
+  bool m_objecter_mclock_service_tracker;
+  bool m_objecter_inject_no_watch_ping;
+  bool m_objecter_debug_inject_relock_delay;
+  bool m_osdc_blkin_trace_all;
+  bool m_objecter_retry_writes_after_first_reply;
+  double m_objecter_tick_interval;
+  double m_objecter_timeout;
+  uint64_t m_osd_client_op_priority;
+  uint64_t m_objecter_inflight_op_bytes;
+  uint64_t m_objecter_inflight_ops;
```

Comment:
what's the point of caching this setting?

nit, i'd suggest remove the `m_` prefix. as most of the member variables of this class don't use this naming convention.